apps/content_filters/filter_condition/filter_condition_field.py

```diff
@@ -150,11 +150,11 @@ def get_value(self, article):
 class FilterConditionGenreField(FilterConditionField):
     def __init__(self, field):
         self.field = FilterConditionFieldsEnum.genre
-        self.entity_name = "genre.name"
+        self.entity_name = "genre.qcode"
```

Comment:
Does this mean the `Genre` `Filter Condition` has not been working for quite some time?

Reply:
Yep, if the genre `name` and its `qcode` is different, then it is not working because whenever we create any filter condition related to genre, it stores `qcode` not `name`.